From 93c2f79425e7def96d8de799a7b7cd7a5bbb2ce9 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Mon, 19 Aug 2024 12:51:25 +0000
Subject: [PATCH] Updating the sepolicy version and sdk version

This patch will update the sepolicy version and sdk
version according the TARGET_RELEASE = ap3a for A15.

Tests Done: Build and boot

Tracked-On: OAM-123546
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 groups/device-specific/caas/compatibility_matrix.xml | 2 +-
 groups/device-specific/caas/manifest.xml             | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/groups/device-specific/caas/compatibility_matrix.xml b/groups/device-specific/caas/compatibility_matrix.xml
index c09add5..ebc5ac8 100644
--- a/groups/device-specific/caas/compatibility_matrix.xml
+++ b/groups/device-specific/caas/compatibility_matrix.xml
@@ -48,7 +48,7 @@
         </interface>
     </hal-->
     <system-sdk>
-        <version>VanillaIceCream</version>
+        <version>35</version>
     </system-sdk>
     <vendor-ndk>
         <version>35</version>
diff --git a/groups/device-specific/caas/manifest.xml b/groups/device-specific/caas/manifest.xml
index 212b8ba..7c5949e 100644
--- a/groups/device-specific/caas/manifest.xml
+++ b/groups/device-specific/caas/manifest.xml
@@ -37,6 +37,6 @@
     </hal>
     <!--<kernel target-level="5"/>-->
     <sepolicy>
-        <version>202504</version>
+        <version>202404</version>
     </sepolicy>
 </manifest>
-- 
2.45.2

